### PR TITLE
Pass env.SCHEMA_BRANCH to contentSchemaDependency

### PR DESCRIPTION
**Pull request by cbaines**
Otherwise, setting the environment value doesn't change anything.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -60,7 +60,7 @@ node('mongodb-2.4') {
     }
 
     stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
+      govuk.contentSchemaDependency(env.SCHEMA_BRANCH)
```
Comment (suzannehamilton) [inline Jenkinsfile]: This looks good, but I discovered the other day that you can use `params.SCHEMA_BRANCH` instead. That means you can also delete the `initializeParameters` call on line 37.

Comment (cbaines) [inline Jenkinsfile]: Ok, I've updated this branch accordingly. I didn't remove the call, but the inclusion of SCHEMA_BRANCH in 9d521bb , can the entire call be removed?

Comment (suzannehamilton) [inline Jenkinsfile]: Yeah, it's safe to remove it entirely. The jenkinslib in govuk-puppet used to call `env.IS_SCHEMA_TEST` but it also uses `params` now.

Comment (cbaines) [inline Jenkinsfile]: Great, I've removed it in  f85e4c8

**Comment by cbaines** [comment]
The tests for frontend are failing at the moment, I've proposed https://github.com/alphagov/frontend/pull/1168 to fix this.

**Pull request merged**